ext/session/php_session.h

```diff
@@ -247,7 +247,7 @@ PHPAPI zend_string *php_session_create_id(PS_CREATE_SID_ARGS);
 PHPAPI int php_session_validate_sid(PS_VALIDATE_SID_ARGS);
 PHPAPI int php_session_update_timestamp(PS_UPDATE_TIMESTAMP_ARGS);
 
-PHPAPI void session_adapt_url(const char *, size_t, char **, size_t *);
+PHPAPI void session_adapt_url(const char *url, size_t urllen, char **new, size_t *newlen);
```

Comment:
Headers need to be compatible with C++, so this should use something other than `new`.